test/validate/windows_validate.go

```diff
@@ -13,12 +13,22 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	cnsWinLabelSelector = "k8s-app=azure-cns-win"
+)
+
 var (
 	hnsEndPointCmd      = []string{"powershell", "-c", "Get-HnsEndpoint | ConvertTo-Json"}
 	hnsNetworkCmd       = []string{"powershell", "-c", "Get-HnsNetwork | ConvertTo-Json"}
 	azureVnetCmd        = []string{"powershell", "-c", "cat ../../k/azure-vnet.json"}
 	azureVnetIpamCmd    = []string{"powershell", "-c", "cat ../../k/azure-vnet-ipam.json"}
 	restartKubeProxyCmd = []string{"powershell", "Restart-service", "kubeproxy"}
+	cnsWinLocalCacheCmd = []string{
```

Comment:
nit: It gives only the assigned IPs ? Not the full local cache as we have a filter. Would good to rename it to something that captures that.

Reply:
Matched the pre-existing naming within `linux_validate.go`, so I will change there as well.

Would `*CachedAssignedIPStateCmd` work?